ProjectSettings/InputManager.asset

```diff
@@ -41,10 +41,10 @@ InputManager:
     m_Name: Vertical1
     descriptiveName: 
     descriptiveNegativeName: 
-    negativeButton: s
-    positiveButton: w
-    altNegativeButton: s
-    altPositiveButton: w
+    negativeButton: w
+    positiveButton: s
```

Comment:
I think this is inverted now. s should be negativeButton and w should be positiveButton.